lib/datasources/invitation.ts

```diff
@@ -53,10 +53,10 @@ export default (_: Request) => ({
           __typename: "Invitation" as const,
           id: i.id,
           status: i.status,
-          email_address: i.emailAddress,
-          created_at: new Date(i.createdAt).toISOString(),
-          updated_at: new Date(i.updatedAt).toISOString(),
-          worker_id: i.publicMetadata?.tendrel_id as string,
+          emailAddress: i.emailAddress,
+          createdAt: new Date(i.createdAt).toISOString(),
+          updatedAt: new Date(i.updatedAt).toISOString(),
+          workerId: i.publicMetadata?.tendrel_id as string,
```

Comment:
forgive my unfamiliarity but wanted to double check this is backwards compatible with whats out in prod rn?

Reply:
definitely not, but there's about zero console traffic at any given time so i'll deploy both to beta tonight